src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -452,6 +454,9 @@ export class ViewConfigItemComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   checkValidityOnChange(key: string, configValue: string) {
+    if (configValue.trim().length === 0) {
+      this.form.controls[key].setErrors({'required': true});
```

Comment:
This is applicable only of config item optional item mandatory is True. Looks we need to have a custom validator overriding  the required validator to handle whitspaces